domains/transfer-agreements/TransferAgreements.API/API/Controllers/ConnectionsController.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Linq;
+using API.ApiModels.Responses;
+using API.Data;
+using API.Extensions;
+using API.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Controllers;
+
+[Authorize]
+[ApiController]
+[Route("api/connections")]
```

Comment:
This requires that a ingress-route is defined in eo-base-environment

Reply:
I did not know that. Will you show me at some point?

Reply:
Yes. Short answer is add to this file k8s\energy-origin-apps\ingress-routes\base\ingress-routes.yaml in eo-base-environment